### PR TITLE
Script improvements

### PR DESCRIPTION
**Pull request by MrZoidberg**
1. Removed pdf from git
2. Optimized Dockerfile build order. Now it won't build dependencies on .py file change
3. Added text wrapping to issue summary that is longer than 50 characters. Otherwise I cannot see all the text in the resulting pdf.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -119,6 +120,7 @@ def cell_value(col, row, date, issue):
     elements.append(p)
 
     doc.build(elements)
+    print 'Done'
```
Comment (jordanjambazov) [inline src/main.py]: Since the Docker image is relying on Python 3, we need to make sure the code is python3 compatible. Please use print function.

**Comment by jordanjambazov** [comment]
Hello @MrZoidberg,

Thanks for your contribution! My only remark is the Python 2 style print function, that breaks compatibility with next version. Please check that out and I will merge the pull request.

Regards,
Jordan

**Comment by MrZoidberg** [comment]
@jordanjambazov sure

**Comment by MrZoidberg** [comment]
done

**Review by jordanjambazov — APPROVED** [review]

**Pull request merged**